src/Components/Search/NewSearch/NewSearchBarInput.tsx

```diff
@@ -182,6 +171,10 @@ const NewSearchBarInput: FC<NewSearchBarInputProps> = ({ relay, viewer }) => {
     redirect(option.href)
   }
 
+  const handleClickOnFooter = () => {
+    setForceRerender(prevCounter => prevCounter + 1)
```

Comment:
One issue that I've introduced in this PR and not aware how to handle it properly: when clicking on the "See full results" item, user is redirected to a /search?term=query page and **dropdown should be closed**.

For some reason, when clicking on the footer component, dropdown area is not being closed (I think close event is not being dispatched in the palette's component). Is there a way to trigger close somehow? Or maybe I should add a new argument to the AutocompleteInput component in palette so that it will be triggering "close" event when clicking on footer? @dzucconi

Reply:
`setForceRerender` looks like a hack; I think any time you're including these state counters we're papering over some other issue.

Reply:
oh, cool, I didn't know this existed! And seems like it was added recently for this particular case: https://github.com/artsy/palette/pull/1296